vision/cloud-client/detect/detect_pdf.py

```diff
@@ -31,7 +31,7 @@
 from google.protobuf import json_format
 
 
-# [START vision_async_detect_document_ocr]
+# [START vision_text_detection_pdf_gcs]
```

Comment:
the segment `pdf` might not be appropriate, since the API also accepts TIFF files.

Reply:
The API might accept TIFF, but this sample demonstrates PDF